content/docs/reference/pkg/nodejs/pulumi/cloudflare/_index.md

```diff
@@ -265,29 +266,6 @@ Provides a Cloudflare Access Policy resource. Access Policies are used
 in conjunction with Access Applications to restrict access to a
 particular resource.
 
-#### Example Usage
```

Comment:
@stack72, any idea why we're losing the examples? I see they were removed in https://github.com/pulumi/pulumi-cloudflare/commit/6b27274e577883691010c24a610868d2cf7c9ee2#diff-54e866f926ca8c6e87f7fdee3af014dcL14. Is this a bug due to bridge changes?

Reply:
@justinvp this may be due to changes in the examples to have hcl2 syntax which t2pulumi doesn't support yet - we had another issue os this elsewhere